glide.yaml

```diff
@@ -1,9 +1,9 @@
 package: k8s.io/cloud-provider-azure
 import:
 - package: k8s.io/kubernetes
-  version: v1.12.0-alpha.0
+  version: v1.13.0-alpha.0
 - package: github.com/Azure/azure-sdk-for-go
-  version: v14.6.0
+  version: v19.0.0
```

Comment:
why change version of `azure-sdk-for-go`?

Reply:
k/k has upgraded to v19.0.0